.github/workflows/php-test.yml

```diff
@@ -12,10 +12,12 @@ jobs:
     strategy:
       matrix:
         php-versions: ['7.3', '7.4']
```

Comment:
Don't `stable21`, `stable22`, `stable23` support `8.0` as well?

Reply:
Absolutely. We should restructure the whole matrix because 8.0 is recommended for most versions of Nextcloud. However, this PR is merely about dropping PHP v7.3 to fix our pipeline.